simuq/aais/heisenberg.py

```diff
@@ -0,0 +1,44 @@
+from simuq.aais.qmachine_factory import QMachineFactory
+from simuq.qmachine import *
```

Comment:
Let's change out the wildcard imports for specific ones; applies to other files too